parser.go

```diff
@@ -116,10 +116,43 @@ func ParseFile(filePath string) (*APIDefinition, error) {
 		return nil, ramlError
 	}
 
+	postProcess(apiDefinition)
```

Comment:
Breaking this down into separate functions is probably unnecessary but there are various references throughout the code to a post-processing stage, so figured it was worthwhile defining this.